tests/unit/executor_tests/test_profile.py

```diff
@@ -35,6 +41,34 @@ def test_profile():
     run(expriment_id, None, None, 2, [])
 
 
+def test_save_json():
+    experiment_id = "test_save_json"
+    environment.init(experiment_id)
+    setup_test_environment()
+    test_dict = {
+        "model_name": "save_json",
+        "image_size_height": 128,
+        "image_size_width": 64,
+        "num_classes": 3,
+        "parameters": "test_node",
+        "flops": "test_flops",
+    }
+    if not os.path.exists(environment.EXPERIMENT_DIR):
+        os.makedirs(environment.EXPERIMENT_DIR)
+
+    _save_json(
+        name=test_dict["model_name"],
+        image_size=(test_dict["image_size_height"], test_dict["image_size_width"]),
+        num_classes=test_dict["num_classes"],
+        node_param_dict=test_dict["parameters"],
+        node_flops_dict=test_dict["flops"]
+    )
+    output_file = os.path.join(environment.EXPERIMENT_DIR, "{}_profile.json".format(test_dict["model_name"]))
+    file_data = json.loads(open(output_file, 'r').read())
```

Comment:
You need to close the file or use `with` block (Sorry, I forgot to point this out... 🙇).

And you can avoid `read()` by using `json.load()` instead of `json.loads()`. It will consume less memory (in this case, it's not a big problem because JSON file is not so big but it's better in general).
```py
with open(output_file, 'r') as fp:
  file_data = json.load(fp)
```